src/main/java/io/castle/client/internal/backend/OkRestApiBackend.java

```diff
@@ -192,16 +195,17 @@ public void onFailure(Call call, IOException e) {
 
             @Override
             public void onResponse(Call call, Response response) throws IOException {
-                Castle.logger.debug("Identify request successful");
+                try (ResponseBody responseBody = response.body()) {
```

Comment:
If we don't use the response body, should we read it? Wouldn't `response.close()` be enough here?

Reply:
It is definitely enough, updating.